### ISSUE TITLE
Textarea fields are introducing <p> tags even when formatting set to 'None'

### ISSUE THREAD

**Issue by andyfoote** [ExpressionEngine/ExpressionEngine#108]
**Description of the problem**
In an ExpressionEngine 5.1.2 I have a requirement where I can paste some HTML into a textarea. The HTML which I paste in should *not* be modified. For this reason I opted *not* to use the Rich Text Editor as we're all aware these can introduce new markup or change markup randomly. I therefore used a Textarea and set the formatting to 'None'. I can paste my HTML fine into this. After it's saved it looks exactly the same as what I pasted in, inside the Admin panel. However, when I try and output it in a template, it is putting `<p>` tags in random places within the markup (these `<p>` tags are not part of the HTML I have input myself). See 'How to reproduce' for an example.

**How To Reproduce**
1. Create a field (Developer > Fields > New Field).
2. Type: Textarea
3. Name: foo
4. Text formatting: None 
5. Create/edit an entry and put the following markup in this field: `<li id="chapter1" class="nav-header"> <a href="#ch1">1. ABC</a> </li> <li id="chapte2" class="nav-header"> <a href="#ch2">2. DEF</a> <ul class="nav nav-list nav-sub-list"> <li class="nav-title"><a href="#ch2_1">2.1 DEF Point 1</a></li> <li class="nav-title"><a href="#ch2_2">DEF Point 2</a></li> </ul> </li> <li id="chapter3" class="nav-header"> <a href="#ch3">GHI</a> </li> <li id="chapter4" class="nav-header"> <a href="#ch4">JKL</a> </li> <li id="chapter5" class="nav-header"> <a href="#ch5">MNO</a> </li>`
6. Observe that in the Admin panel, the markup looks exactly as it's shown above - there are no `<p>` tags anywhere in that block of HTML.
7. Output the field within a template: `{foo}`
8. The content which is output contains random `<p>` tags: ` <p><li id="chapter1" class="nav-header"> <a href="#ch1">1. ABC</a> </li> <li id="chapte2" class="nav-header"> <a href="#ch2">2. DEF</a> </p><ul class="nav nav-list nav-sub-list"> <li class="nav-title"><a href="#ch2_1">2.1 DEF Point 1</a></li> <li class="nav-title"><a href="#ch2_2">DEF Point 2</a></li> </ul> </li> <li id="chapter3" class="nav-header"> <a href="#ch3">GHI</a> </li> <li id="chapter4" class="nav-header"> <a href="#ch4">JKL</a> </li> <li id="chapter5" class="nav-header"> <a href="#ch5">MNO</a> <p></li></p>`

**Error Messages**
There are no error messages shown

**Screenshots / Videos / Template Code**
This is how it looks in the Admin panel, even after pressing Save. It is exactly the same as the markup entered during step (5) above. But obviously different to what EE is outputting in the template during (8):
<img width="1152" alt="screenshot 2019-01-25 at 11 21 28" src="https://user-images.githubusercontent.com/35595432/51743048-6f3c1280-2093-11e9-955c-c08b8311df4e.png">

**Environment Details:**
 - Version: 5.1.2
 - PHP Version:  7.0.33
 - MySQL Version: 5.5.60 MariaDB
 - OS: MacOS
 - Web Server: CentOS, Apache

**Possible Solution**
If you look in the database at the appropriate `exp_channel_data_field_` table the markup is saved as per (5).  I noticed that the `field_ft_` column in the above database table is set to `xhtml` even though I put 'None' as the formatting option at step (4). If I manually change that column from `xhtml` to `none` in the database it works fine. So I guess EE isn't saving the user's preference from step (4) correctly?

**Additional context**
n/a

**Comment by andyfoote** [comment]
If this is expected behaviour please clarify how a user could store HTML inside an EE field without any modification taking place. This is a very basic requirement - HTML entered by the user should remain intact without any modification by EE or a Rich Text Editor.

**Comment by andyfoote** [comment]
I can't reproduce it. One thing I do recall is that I'd tried to change my fields between Rich Text Editor to Textarea. Then I changed them back to Rich Text Editor. However, the radio button for 'Formatting' on the RTE was definitely set to 'None' in the Admin panel. But then when I looked in the `field_ft_x` column it said `xhtml`. I tried to resave the field to see if it updated the database, but it didn't. My workaround was to use Navicat (a MySQL administration tool similar to phpMyAdmin) and manually change `xhtml` to `none`.
